pages/members/[id]/index.js

```diff
@@ -1,10 +1,45 @@
+import { getDataURL, getImgURL } from 'helper-functions/urls';
+import PropTypes from 'prop-types';
+import fetch from 'cross-fetch';
 import React from 'react';
-import { useRouter } from 'next/router';
 import Profile from 'components/member-profile';
+import NotFound from 'components/404page';
```

Comment:
NIT: `not-found-page`, than calling it `404page`

Reply:
@ankushdharkar Done, Please check